lalrpop-test/src/lib.rs

```diff
@@ -20,7 +20,6 @@ macro_rules! lalrpop_mod_test {
     ($(#[$attr:meta])* $vis:vis $modname:ident) => {
         lalrpop_mod!(
             #[allow(clippy::ptr_arg)]
```

Comment:
In doc/parcal/lalrpop/src/main.rs, both clippy lints(vec_box and ptr_arg) were removed. Can the same be done here?